api/swagger.yaml

```diff
@@ -4741,6 +4741,11 @@ definitions:
         properties:
           NetworkMode:
             type: "string"
+          Annotations:
+            description: "Arbitrary key-value metadata attached to container"
```

Comment:
This probably needs `x-nullable: true` as well

Reply:
Yes, thanks. Added.